src/components/graph-view.js

```diff
@@ -80,6 +80,7 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
     canSwapEdge: () => true,
     canDeleteSelected: () => true,
     allowMultiselect: true,
+    allowCopyEdges: false,
```

Comment:
Can this be an object of `CopyPasteOptions` instead of boolean?

Reply:
We had originally discussed this for reducing clutter for future milestones but with the latest ERD I do not anticipate any additional modifications to be required to `react-digraph` so I'm not sure if this is necessary. The only possible modification would be for a feature that's been determined to be out-of-scope, which would only require one additional flag, so at best the object would only be holding 2 booleans.